docs/guide/built-in-rules-nested.md

```diff
@@ -188,7 +188,7 @@ $result = (new Validator())->validate($data, $rule);
 $errors = $result->getErrorMessagesIndexedByPath();
 ```
 
-The contents of the errors will be:
+The content of the errors will be:
```

Comment:
https://dictionary.cambridge.org/grammar/british-grammar/content-or-contents

The errors are countable.